pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MergeRollupMinionClusterIntegrationTest.java

```diff
@@ -178,6 +187,15 @@ private TableTaskConfig getMultiLevelConcatTaskConfig() {
     return new TableTaskConfig(Collections.singletonMap(MinionConstants.MergeRollupTask.TASK_TYPE, tableTaskConfigs));
   }
 
+  private SegmentPartitionConfig getMultiColumnSegmentPartitionConfig() {
+    Map<String, ColumnPartitionConfig> columnPartitionConfigMap = new HashMap<>();
+    ColumnPartitionConfig columnOneConfig = new ColumnPartitionConfig("murmur", 1);
+    columnPartitionConfigMap.put("AirlineID", columnOneConfig);
+    ColumnPartitionConfig columnTwoConfig = new ColumnPartitionConfig("murmur", 1);
+    columnPartitionConfigMap.put("Month", columnTwoConfig);
+    return new SegmentPartitionConfig(columnPartitionConfigMap);
```

Comment:
Multi-columns partitioning config. This config is used by `MergeRollupTaskExecutor` (`SegmentProcessFramework`) to partition the data.